docs(fix): removed broken images MTA-1604 (#245)
scaleway/docs-content: dedibox/dedicated-servers/quickstart.mdx

   </Message>

> 4. In the side menu, under **Connection** navigate to the **Auth** sub-category. Do this by expanding **Connection**, then **SSH** and finally clicking on **Auth**.

Reviewer: ```suggestion
4. Use the side menu to navigate to the **Auth** sub-category by expanding **Connection**, then **SSH** and finally clicking on **Auth**.
```
